frontend/src/Components/ContentComponent.tsx

```diff
@@ -128,12 +126,53 @@ function updateImg(img: HTMLImageElement) {
     }
 }
 
-function updateSpoiler(spoiler: HTMLSpanElement) {
-    const spoilerOnClickHandler = () => {
-        spoiler.classList.remove('spoiler');
-        spoiler.removeEventListener('click', spoilerOnClickHandler);
+function updateSpoilers(div: HTMLDivElement) {
+    const spoilers = div.querySelectorAll('span.spoiler');
+    if (spoilers.length === 0) {
+        return;
+    }
+
+    const svg = document.getElementById('sharpBlurSVG');
```

Comment:
Perhaps we can have this SVG in file form in `Assets/` folder and always added to body?


e.g. import svg like this:
https://github.com/spaceshelter/orbitar/blob/1ed9e95a34678ac7f02620c9923a901a7eff2fa0/frontend/src/Components/Topbar.tsx#L9

and add it somewhere here: 
https://github.com/spaceshelter/orbitar/blob/5580fd1f62a2f3d55e10aa51e5da0d3f70fa644a/frontend/src/App.tsx#L110

Reply:
Yes, this was the first version, but this svg is not a image, Im not sure Assets/ is the correct place, and decided to create the svg on demand. But if its ok, I can move svg-filter to assets/

Reply:
The definition of "asset" is wider then just an image. I think it would be more readable to have this svg the explicit form.